llama-index-integrations/tools/llama-index-tools-box/llama_index/tools/box/ai_extract/base.py

```diff
@@ -0,0 +1,54 @@
+from llama_index.core.schema import Document
+from llama_index.core.tools.tool_spec.base import BaseToolSpec
+
+from box_sdk_gen import (
+    BoxClient,
+)
+
+from llama_index.readers.box.BoxAPI.box_api import (
+    box_check_connection,
+    get_box_files_payload,
+    get_files_ai_extract_data,
+)
+
+
+class BoxAIExtractToolSpec(BaseToolSpec):
+    spec_functions = ["ai_extract"]
+
+    _box_client: BoxClient
+
+    def __init__(self, box_client: BoxClient) -> None:
+        self._box_client = box_client
+
+    def ai_extract(
```

Comment:
Please add a nice doc string for this spec_function

Reply:
added